controllers/rabbitmq/rabbitmq_controller.go

```diff
@@ -356,6 +362,31 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ct
 
 	if clusterReady {
 		instance.Status.Conditions.MarkTrue(condition.DeploymentReadyCondition, condition.DeploymentReadyMessage)
+		// Let's wait DeploymentReadyCondition=True to apply the policy
+		queue := instance.Spec.QueueType
+		ismirrored := instance.Status.QueueType
+
+		if queue == "Mirrored" && *instance.Spec.Replicas > 1 {
+			if ismirrored != "Mirrored" {
```

Comment:
probably no need for a nested if. could be part of the above.

Reply:
ouch right